nfc_ros/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>nfc_ros</name>
+  <version>0.0.0</version>
+  <description>The nfc_ros package</description>
+
+  <maintainer email="sktometometo@todo.todo">sktometometo</maintainer>
+
+  <license>TODO</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>catkin_virtualenv</build_depend>
+  <build_export_depend>message_generation</build_export_depend>
+  <exec_depend>message_runtime</exec_depend>
```

Comment:
You need to add `python-libusb1`.
you will need to add python-libsub to rosdistro
https://github.com/ros/rosdistro